analytics/views.py

```diff
@@ -1,3 +1,30 @@
 # from django.shortcuts import render
+from django.utils.decorators import decorator_from_middleware
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from . import middleware, models, serializers
 
 # Create your views here.
+
+
+@api_view(['GET'])
+def home(_request):
+    """Render the most basic home page ever designed."""
+    return Response('Analytics app seems to be running.')
+
+
+@api_view(['GET', 'POST'])
+@decorator_from_middleware(middleware.AnalyticsTokenAuthMiddleware)
```

Comment:
https://docs.djangoproject.com/en/4.1/topics/http/middleware/#activating-middleware jeśli middleware ma być wszędzie to dlaczego nie jest wpisany do settingsów?

Reply:
Zrobiłem to celowo, w sumie treść wskazywała tylko na /api/events. Ale to było nieco niejasne - i skoro sugestia jest taka, to 3a189efe38fe9aa81b8544c7c2f64c64be86ff08. Po to w ogóle napisałem to jako middleware, żeby dało się łatwo zmieniać jego sposób użycia.